### DISCUSSION TITLE
Textures not loading with Three.js

### DISCUSSION THREAD

**Discussion by bongoslav** [vercel/community#2018] [Help] [answered]
### Summary

When I deploy my HTML/CSS/JS app that is using textures for Three.js the textures are not rendered and Vercel cannot find them with the provided path.
This is my folder structure:
.
├── api.js
├── dist
│   ├── assets
│   │   └── index-9ea7e444.js
│   └── index.html
├── index.html
├── main.js
├── package.json
├── package-lock.json
├── README.md
└── textures
    ├── earth.jpg
    └── stars.jpg

Running it locally with Vite I have no issues. I am referencing my images with:
    const texture = new THREE.TextureLoader().load("./textures/earth.jpg");
    
I tried moving them to ./public/static/images/  but I have the same issue.

### Additional information

Here is the deployed app: https://earth-earthquakes-live.vercel.app/ .

**Comment by iaruso** [comment, marked as answer]
I hope this is the solution for you. I tried here and it worked.

1. Create a public folder
2. Inside of it create static folder
3. Then another folder named images
4. Transfer those textures images to that folder and then delete textures folder
5. In your code replace with "./static/images/earth.jpg" and "./static/images/stars.jpg"

If it works, please accept my answer.

**Reply by bongoslav** [reply]
thanks, it worked!
